pkg/placement/tables.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package placement
+
+type PlacementTables struct {
+	HostList     []*HostInfo `json:"host_list,omitempty"`
+	TableVersion uint64      `json:"table_version,omitempty"`
+}
+type HostInfo struct {
+	Name      string   `json:"name,omitempty"`
+	AppID     string   `json:"app_id,omitempty"`
+	Entities  []string `json:"entities,omitempty"`
+	UpdatedAt int64    `json:"updated_at,omitempty"`
+}
+
+// GetPlacementTables returns the current placement host infos.
+func (p *Service) GetPlacementTables() (*PlacementTables, error) {
+	m := p.raftNode.FSM().State().Members()
+	version := p.raftNode.FSM().State().TableGeneration()
+	response := &PlacementTables{
+		TableVersion: version,
+	}
+	members := make([]*HostInfo, 0, len(m))
```

Comment:
Can you change this to not be a pointer? Since we don't need null values here, I believe it should be more efficient


```suggestion
	members := make([]HostInfo, 0, len(m))
```